server/src/visitors/YmlClassVisitor.ts

```diff
@@ -22,6 +27,14 @@ export class YmlClassVisitor extends YmlToBdlBaseVisitor {
       CompletionItemKind.Method,
       this.classId,
     );
+    this.definitions.addDefinition(
```

Comment:
You seem to be always calling this `addDefinition` method along with a `createLocation` call. Why not change `addDefinition`'s parameter type and accept this `node` thingy and an URI? :thinking:

`addDefinition(node, this.uri);`

Reply:
I will try with the node only. Good advice.

Reply:
Unfortunately, after doing that I had cross-references problem that made execution fails.